client/src/components/utils/PopUp/style.css

```
@@ -0,0 +1,62 @@
+.popup{
+  position: absolute;
+  text-align: center;
+  width: 80%;
```

`Suggestion` you can use vw vh 
```suggestion
  width: 80vw;
```

---

-_-